src/node_main.cc

```diff
@@ -55,6 +55,18 @@ int main(int argc, char *argv[]) {
   // calls elsewhere in the program (e.g., any logging from V8.)
   setvbuf(stdout, nullptr, _IONBF, 0);
   setvbuf(stderr, nullptr, _IONBF, 0);
+#ifdef _AIX
+  // AIX passes the same argv array that ps and other utilities see.
```

Comment:
"use" might be better - on the other had it does make it clear this isn't private data. It's visible from outside of the node process.